calibr8/tests.py

```diff
@@ -516,7 +600,7 @@ def test_assert_version_match(self):
     @pytest.mark.parametrize("residual_type", ["relative", "absolute"])
     def test_plot_model(self, residual_type):
         em = _TestPolynomialModel(independent_key='S', dependent_key='A365', mu_degree=1, scale_degree=1)
-        em.theta_fitted = [0, 2, 0.1, 1]
+        em.theta_fitted = [0, 2, 0.1, 1, 2]
```

Comment:
Can you please explain why all theta have an entry more? Is this the bug you mentioned?

Reply:
Yes, the number I added is the degree of freedom. All these tests passed the incorrect number of parameters.

Reply:
:O